src/MainWindow/main_window.cpp

```diff
@@ -371,7 +371,12 @@ void MainWindow::createActions() {
   });
 }
 
-void MainWindow::gui_start() { showWelcomePage(); }
+void MainWindow::gui_start() {
+  if (GlobalSession->CmdLine()->Script().empty())
+    showWelcomePage();
```

Comment:
Can we also record the user selection to disable it on start-up if one chooses too? We can do this separately.

Reply:
@TarasSydor is this resolved ?